src/components/MenuItem.tsx

```diff
@@ -178,6 +178,9 @@ type MenuItemBaseProps = {
     /** Text that appears above the title */
     label?: string;
 
+    /** Limit to truncate description for menu item */
+    limit?: number;
```

Comment:
I feel like this is more explicit. 
```suggestion
    characterLimit?: number;
```